deltaspike/modules/data/impl/src/main/java/org/apache/deltaspike/data/impl/meta/RepositoryComponent.java

```diff
@@ -201,24 +201,27 @@ private void initialize()
         }
     }
 
-    private Set<Class<?>> collectClasses()
-    {
+    private Set<Class<?>> collectClasses() {

```

Comment:
Could you add the { on the next line, adhering to our coding standards? I would imagine the checkstyle checks fail with this change.